tests/main/refresh-all-undo/task.yaml

```diff
@@ -23,6 +23,17 @@ prepare: |
     echo "And the daemon is configured to point to the fake store"
     setup_fake_store $BLOB_DIR
```

Comment:
Indeed, fantastic find! I second mvo's suggestion, it is easier to get things right this way.